acdh-schema.owl

```diff
@@ -212,14 +242,26 @@ Kardinalität von acdh:hasCategory für acdh:BinaryContent gändert zu 0-n, um m
 
     <owl:AnnotationProperty rdf:about="https://vocabs.acdh.oeaw.ac.at/schema#defaultValue">
         <rdfs:comment xml:lang="de">Gibt den Standardwert einer Eigenschaft an. Der Standardwert wird verwendet, wenn diese Eigenschaft in einer Klasse verwendet wird, aber kein Wert angegeben wurde.</rdfs:comment>
-        <rdfs:comment xml:lang="en">Indicates a default property value. The default value is assigned when a class matches property&apos;s domain but no property value has been provided.</rdfs:comment>
+        <rdfs:comment xml:lang="en">Indicates a default property value. The default value is assigned when a class matches property&amp;apos;s domain but no property value has been provided.</rdfs:comment>
```

Comment:
I think there is a small typo here, `&amp;apos;` should be corrected to `&apos;`